tests/unit/cocoon/test_utilities.py

```diff
@@ -1095,8 +1099,25 @@ def test_handle_nested_default_and_column_mapping_success(
 
     @parameterized.expand(
         [
+            # Test empty string
+            [
+                "Test empty string",
```

Comment:
I think it would be better to pull the DataFrame construction out into a separate parameterised function to make it easier to read the test cases, not a blocker for this PR but maybe raise a Mira